mm/MemOps.c

```
@@ -276,442 +56,27 @@ void* FastMemcpy(void* restrict dest, const void* restrict src, uint64_t size) {
-                "movdqa %%xmm0, 32(%0)\n"
-                "movdqa %%xmm0, 48(%0)\n"
-                "movdqa %%xmm0, 64(%0)\n"
-                "movdqa %%xmm0, 80(%0)\n"
-                "movdqa %%xmm0, 96(%0)\n"
-                "movdqa %%xmm0, 112(%0)\n"
-                :
-                : "r"(p + i)
-                : "memory"
-            );
-        }
+    const CpuFeatures * features = GetCpuFeatures();
```

Clang (14.0.6)

[warning] variable 'features' is not initialized

(cppcoreguidelines-init-variables)